CMakeLists.txt

```diff
@@ -373,6 +373,10 @@ else ()
     message (STATUS "Set jemalloc narenas ${JEMALLOC_NARENAS}")
 endif ()
 
+if (ENABLE_JEMALLOC)
+    add_compile_definitions(WITH_JEMALLOC)
+endif ()
```

Comment:
Do we need to add `WITH_JEMALLOC` for all files?

Reply:
let me check, maybe USE_JEMALLOC is enough

Reply:
fixed